docs/data/data-grid/getting-started/getting-started.md

```diff
@@ -6,13 +6,7 @@
 
 Using your favorite package manager, install `@mui/x-data-grid-pro` or `@mui/x-data-grid-premium` for the commercial version, or `@mui/x-data-grid` for the free community version.
```

Comment:
To consider expanding on the reason for using the tag.

```suggestion
Using your favorite package manager, install `@mui/x-data-grid-pro` or `@mui/x-data-grid-premium` for the commercial version, or `@mui/x-data-grid` for the free community version.

Note we're using the `next` tag to download the latest beta version of v6.
```

Reply:
Done!
<img width="847" alt="Screenshot 2023-02-03 at 13 58 21" src="https://user-images.githubusercontent.com/13808724/216609489-3bc5fd80-c989-4e95-8b10-bf20c7138410.png">